pkg/server/container_stats.go

```diff
@@ -25,6 +25,8 @@ import (
 
 // ContainerStats returns stats of the container. If the container does not
 // exist, the call returns an error.
+// TODO(windows): hcsshim Stats is not implemented, add windows support after
```

Comment:
Ah. I have a PR for you. Will link you in

Reply:
Thanks! I'll file an issue after my next PR, and will add the support once the hcsshim side is ready. :)